### ISSUE TITLE
checkField ignoring null values

### ISSUE THREAD

**Issue by BerkOFF** [DanielCardonaRojas/verify#2]
It's undocumented that checkField is ignoring running validators in case the null value is returned by the selector.

**Comment by luqmanhakem** [comment]
Yup true. I've verified this as my breakpoints seems to not hit it.

Looking forward for this issue to be resolve
